bankroll/model/modelconverter.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+from typing import List
+
+from bankroll.model import ConvertableModel, Position, Trade
+
+
+class ModelConverter:
+    @classmethod
+    def dataframe(cls, items: List[ConvertableModel]) -> pd.DataFrame:
+        if len(items) > 0:
+            firstItem = items[0]
+            assert all(isinstance(i, type(firstItem)) for i in items)
```

Comment:
I understand what this check is doing, but why is it needed?

Reply:
The columns only make sense for the specific Trade or Position classes, not both.